test/deps.sh

```diff
@@ -9,6 +9,7 @@ set -e
 test () {
   echo "## TESTING DEPENDENCY: $1"
   pushd node_modules/$1
+  npm install --only=dev
```

Comment:
I've added this line to ensure that we have the correct devdeps for running tests.